src/vm/arm64/pinvokestubs.S

```diff
@@ -89,25 +89,86 @@ LOCAL_LABEL(\__PInvokeStubFuncName\()_0):
 // ------------------------------------------------------------------
 // IN:
 // InlinedCallFrame (x0) = pointer to the InlinedCallFrame data, including the GS cookie slot (GS cookie right 
-//                          before actual InlinedCallFrame data)
+//                         before actual InlinedCallFrame data)
 //
 //
-    LEAF_ENTRY JIT_PInvokeBegin, _TEXT
-        // Not yet supported
-        EMIT_BREAKPOINT
-        ret lr
-    LEAF_END JIT_PInvokeBegin, _TEXT
+    NESTED_ENTRY JIT_PInvokeBegin, _TEXT, NoHandler
+    
+        PROLOG_SAVE_REG_PAIR_INDEXED   fp, lr, -32
+        PROLOG_SAVE_REG    x19, 16           //the stack slot at sp+24 is empty for 16 byte alligment
+
+        PREPARE_EXTERNAL_VAR s_gsCookie, x9
+        ldr     x9, [x9]
+        str     x9, [x0]
+        add     x19, x0, SIZEOF__GSCookie
+
+        // x19 = pFrame
+        
+        // set first slot to the value of InlinedCallFrame::`vftable' (checked by runtime code)
+        PREPARE_EXTERNAL_VAR C_FUNC(_ZTV16InlinedCallFrame+16), x9
```

Comment:
The +16 should be outside of the C_FUNC. It doesn't matter functionally, but semantically, the C_FUNC generares a platform specific C function symbol from a platform agnostic one. So in fact, the C_FUNC should not be used for _ZTV16InlinedCallFrame at all, as it is not a function.